dns/zone-configs/k8s.io._0_base.yaml

```diff
@@ -326,16 +326,22 @@ submit-queue:
 testgrid:
   - type: A
     values:
-    - 216.239.32.21
-    - 216.239.34.21
-    - 216.239.36.21
-    - 216.239.38.21
+      - 216.239.32.21
+      - 216.239.34.21
+      - 216.239.36.21
+      - 216.239.38.21
+    # - 34.120.51.46 // TODO(michelle192837): Use this once certificate is active.
   - type: AAAA
     values:
-    - 2001:4860:4802:32::15
-    - 2001:4860:4802:34::15
-    - 2001:4860:4802:36::15
-    - 2001:4860:4802:38::15
+      - 2001:4860:4802:32::15
+      - 2001:4860:4802:34::15
+      - 2001:4860:4802:36::15
+      - 2001:4860:4802:38::15
```

Comment:
I think we still need to put this in quotes ?

Reply:
Done, thanks! ^^